core/gui/src/app/dashboard/component/user/user-dataset/user-dataset-explorer/user-dataset-version-filetree/user-dataset-version-filetree.component.html

```diff
@@ -18,6 +20,11 @@
           nzType="file"
           nzTheme="outline"></i>
         {{ node.data.name }}
+        <span
+          *ngIf="!node.data.children"
+          class="file-size">
+          {{ formatSize(node.data.size) }}
```

Comment:
When I create the new version of a dataset, the `delete` icon appear to be very small:
<img width="482" alt="Screenshot 2024-10-13 at 8 48 05 AM" src="https://github.com/user-attachments/assets/0fb6a980-6e48-4749-8428-573d4636851c">

Also I think maybe we don't need to display the file size at file tree. 
<img width="274" alt="Screenshot 2024-10-13 at 8 49 09 AM" src="https://github.com/user-attachments/assets/4a9b37d5-c780-4b38-bf0e-767a57fa4a59">

display at each file's title is good enough

Reply:
I will remove displaying the file size at file tree.